background.js

```diff
@@ -1,5 +1,6 @@
 import stats from "./module/stats.js"
 import app from "./module/app.js"
+import { storageKey, bubbleCountKey } from './config.js';
```

Comment:
Let's try not use semicolon in this application as a standard 🙂 
```suggestion
import { storageKey, bubbleCountKey } from './config.js'
```